### PR TITLE
Add content type in content form breadcrumb

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 1]
```diff
@@ -123,23 +123,27 @@ class ContentRouter extends OrchestraRouter
...
-        this._displayLoader(Application.getRegion('content'));
-        let url = Routing.generate('open_orchestra_backoffice_content_form', {
-            contentId: contentId,
-            language: language,
-            version: version
-        });
         let contentType = new ContentType();
-        let content = new Content({id: contentId});
-
         $.when(
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: when is useless with one fetch you can use sucess calback

**Comment by alavieille** [comment]
you forget manage version route

**Comment by ngilain** [comment]
@alavieille :update done

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 2]
```diff
@@ -71,25 +71,33 @@ class ContentRouter extends OrchestraRouter
...
-                contentId: contentId
-            },
-            success: (contentVersions) => {
-                let contentVersionsView = new ContentVersionsView({
-                    collection: contentVersions,
-                    settings: {
-                        page: page,
-                        pageLength: Application.getConfiguration().getParameter('datatable').pageLength
+        let contentType = new ContentType();
+        contentType.fetch({
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: use $.when

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 3]
```diff
@@ -123,35 +131,41 @@ class ContentRouter extends OrchestraRouter
...
-                    Application.getRegion('content').html(contentFormView.render().$el);
+                this._displayLoader(Application.getRegion('content'));
+                let url = Routing.generate('open_orchestra_backoffice_content_form', {
+                    contentId: contentId,
+                    language: language,
+                    version: version
+                });
+
+                let content = new Content({id: contentId});
+                $.when(
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: when is useless here

**Comment by ngilain** [comment]
@alavieille : update done

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 4]
```diff
@@ -24,27 +24,83 @@ class ContentRouter extends OrchestraRouter
...
+        this._cachedContentTypes = {};
+
+        this._breadcrumbItems = [
             {
                 label: Translator.trans('open_orchestra_backoffice.menu.contribution.title')
             },
             {
                 label: Translator.trans('open_orchestra_backoffice.menu.contribution.content'),
-                link: '#'+Backbone.history.generateUrl('showContentSummary')
+                link: '#content/summary'
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: create a variable

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 5]
```diff
@@ -24,27 +24,83 @@ class ContentRouter extends OrchestraRouter
...
+     */
+    _cacheContentType(contentType) {
+        this._cachedContentTypes[contentType.get('content_type_id')] = contentType;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    getBreadcrumb(args, name) {
+        let contentTypesRoute = ['newContent', 'editContent', 'manageVersionsContent'];
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: forget listContent

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 6]
```diff
@@ -24,27 +24,83 @@ class ContentRouter extends OrchestraRouter
...
+    }
+
+    /**
+     * Get breadcrumb with Content Type
+     * @param {string} contentTypeId
+     * @returns {Object}
+     */
+    _getBreadcrumbWithContentType(contentTypeId) {
+        let contentType = this._getCachedContentType(contentTypeId);
+        if (typeof contentType === 'object') {
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: object ? why not Content

**Comment by ngilain** [comment]
@alavieille : update done

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js [hunk 7]
```diff
@@ -65,37 +122,36 @@ class ContentRouter extends OrchestraRouter
...
+            apiContext: 'list-version',
+            urlParameter: {
                 language: language,
-                contentType: contentType,
-                siteLanguages: Application.getContext().siteLanguages
-            });
-            Application.getRegion('content').html(contentVersionsView.render().$el);
+                contentId: contentId
+            },
+            success: (contentType) => {
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Content/ContentRouter.js]: contentType ?

**Comment by alavieille** [comment]
:+1:

**Pull request merged**